src/NHibernate.Test/DriverTest/OdbcDriverFixture.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Collections;
+using NHibernate.Dialect;
+using NHibernate.Driver;
+using NHibernate.Engine;
+using NUnit.Framework;
+
+namespace NHibernate.Test.DriverTest
+{
+	[TestFixture]
+	public class OdbcDriverFixture : TestCase
```

Comment:
Copy paste of `SqlClientDriverFixture` which was executed for ODBC too, but with some specific test asserting ignore when it was ODBC. Now separated in its own fixture, with only non-ignored tests.